app/views/shared/_webchat_popup.html.erb

```diff
@@ -1,4 +1,4 @@
-<div aria-hidden="true" class="mobile-webchat__container mobile-webchat--closed" data-dough-component="ChatPopup">
+<div aria-hidden="true" class="mobile-webchat__container mobile-webchat--closed <% if pensions_and_retirement_page? %>display-flex<% end %>" data-dough-component="ChatPopup">
```

Comment:
In fact the whole `mobile-webchat__container` class is rendered obsolete by this change. Maybe that just needs to be called `webchat__container` and removed from view on mobile except where the class is also `pensions` or some-such, which can replace the `display-flex` class. (See also above on the `display-flex` class).